docs/ehrql-new-tutorial-1a.md

```diff
@@ -1,56 +1,36 @@
-# ehrQL tutorial: A minimal dataset definition
+# ehrQL tutorial Part 1: Minimal dataset definition
+By the end of this tutorial, you should able:
 
----8<-- 'includes/data-builder-danger-header.md'
+* to write a very simple dataset definition
+* to run that dataset definition with Data Builder
 
-## Example dataset definition 1a: A minimal dataset definition
+## Full Example
 
-### Learning objectives
-
-By the end of this tutorial, you should know how to:
-
-* Write a very simple dataset definition
-* Run that dataset definition with Data Builder
-
-### The dataset definition we will work with
-
-This is a minimal, but still valid, dataset definition:
-
-!!! todo
-
-    May need to fix up how this code is included.
-    Can we have code annotations?
-    What's the best way of jumping between input, output and dataset
-    definition? Maybe that's a case for having one example per tutorial
-    page?
+We start with a minimal dataset definition. This
+finds the patients whose year of birth is 2000 or later.
 
 ???+ example "Dataset definition: `1a_minimal_dataset_definition.py`"
 
     ```python
     ---8<-- "databuilder/ehrql-tutorial-examples/1a_minimal_dataset_definition.py"
     ```
 
-It finds the patients whose year of birth is 2000 or later.
+If we run this against the sample data provided (see below), it will
+pick out only patients who were born in 2000 or later. 
 
-### The `minimal` data source
-
-???+ example "Data table: `minimal/patients.csv`"
+???+ example "Original Data: `minimal/patients.csv`"
 
     {{ read_csv('databuilder/ehrql-tutorial-examples/example-data/minimal/patients.csv') }}
 
-### Dataset definition 1a output
-
-!!! todo
-
-    Do we need to clarify that the filename corresponds to the outputs already created?
-    And that you'll overwrite these if you use this as a filename?
+In this case, patient 2, 5 and 7. 
```

Comment:
thought: This point is slightly disconnected from the the preceding paragraph to which it relates. Maybe restating something like:

> This dataset definition selects patients 2, 5, and 7 from the data.